ibm/service/kubernetes/data_source_ibm_container_cluster_config.go

```diff
@@ -88,6 +88,12 @@ func DataSourceIBMContainerClusterConfig() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 			},
+			"endpoint_type": {
+				Description: "It can specify what kind of server URL will be used for the cluster context",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     false,
```

Comment:
How can we have Default value as "false" as we know the supported are "private", "link", "vpe"

If we don't have any Default value how does the SDK handles

Reply:
Yes, this is my bad. The default shall be empty string. If it is empty, the SDK won't set he `endpoint` field in the request. Thanks!

Reply:
We don't need to explictly set Default = "" by default in Terraform string will be empty

Reply:
Yes, I have removed this whole line! Thanks!